WordPress/Classes/ViewRelated/Notifications/Views/NoteBlockActionsTableViewCell.swift

```diff
@@ -156,25 +156,25 @@ class NoteBlockActionsTableViewCell: NoteBlockTableViewCell {
     /// Returns the accessibility label for the Approve Button
     ///
     fileprivate var approveAccesibilityLabel: String {
-        return isApproveOn ? Approve.selectedTitle : Approve.normalTitle
+        return isApproveOn ? ApproveComment.TitleStrings.selected : ApproveComment.TitleStrings.approve
     }
 
     /// Returns the accessibility hint for the Approve Button
     ///
     fileprivate var approveAccesibilityHint: String {
-        return isApproveOn ? Approve.selectedHint : Approve.normalHint
+        return isApproveOn ? ApproveComment.TitleHints.selected : ApproveComment.TitleStrings.approve
```

Comment:
I think `ApproveComment.TitleStrings.approve` should be `ApproveComment.TitleHints.approve`?
(TitleString -> TitleHints)